### DISCUSSION TITLE
✅ 🔗 Adaptive text with css clamp (Duplicate of #2219)

### DISCUSSION THREAD

**Discussion by tekonemedia** [elementor/elementor#16913] [Feature Request]
It would be really nice to have % option for text size so we can set the text size to percentage of container width. especially with so many ultrawide monitor users now adays.

Also would be great to have a min/max text size option for use with vw/% text size values. 

This can already be done in Webflow.. Just saying..

**Comment by tekonemedia** [comment]
css clamp would work great for this. Clamp is now supported on basically every browser under the sun.

**Comment by arielk** [comment]
Ref: https://github.com/elementor/elementor/issues/2219
